examples/enterprise-azure-deployment/enterprise-workspace-deploy/main.tf

```diff
@@ -0,0 +1,91 @@
+provider "azurerm" {
```

Comment:
please let's discuss internally how we do manage examples :)

Reply:
Sounds good.  Lol, yeah, this example is specific to setting up on azure.  Let me drop that and think about a better enterprise example.  This is more of an end-to-end anyway which would have to call the specific cloud provider to setup the workspace in the first place.